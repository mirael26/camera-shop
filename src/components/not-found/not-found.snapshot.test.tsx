import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import NotFound from './not-found';

test('NotFound renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<NotFound/>);
  expect(asFragment()).toMatchSnapshot();
});
