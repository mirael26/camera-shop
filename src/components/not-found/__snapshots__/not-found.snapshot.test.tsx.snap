// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<DocumentFragment>
  <section
    class="not-found"
    data-testid="not-found"
  >
    <div
      class="container"
    >
      <h1
        class="title title--h2 not-found__title"
      >
        404 Not Found
      </h1>
      <a
        class="btn btn--purple"
        href="/"
      >
        Вернутся на главную
      </a>
    </div>
  </section>
</DocumentFragment>
`;
